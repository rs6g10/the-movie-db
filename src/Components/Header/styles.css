.header {
    margin: 30px 0;
    padding: 0;
}

.logo {
    width: 100%;
    max-width: 60px;
    margin: 0 auto;
    margin-bottom: 25px;
    display: block;
}

@media (min-width: 576px) {
    .logo {
        display: inline-block;
        margin: 0;
    }
}