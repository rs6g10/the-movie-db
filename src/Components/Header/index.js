import React from 'react';
import { Link } from 'react-router-dom';
import Searchbox from '../Searchbox';
import './styles.css';

const TMDBLogo = 'https://www.themoviedb.org/static_cache/v4/logos/stacked-green-cae7a95e2590dbdde28284ac26245cb2792788838f5c498b892e8d01c183e6f3.svg';

const Header = () => (
  <div className="col-xs-12 header">
    <div className="row">
      <div className="col-xs-12 col-sm-6 col-lg-3">
        <Link to="/">
          <img src={TMDBLogo} className="logo" alt="The Movie Database" />
        </Link>
      </div>
      <div className="col-xs-12 col-sm-6 col-lg-9">
        <Searchbox
          placeHolder="Search"
          showResults
          apiKey="mysecretkey"
        />
      </div>
    </div>
  </div>
);

export default Header;
