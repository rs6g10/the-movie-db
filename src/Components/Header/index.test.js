import '../../test/setupEnzyme';

import React from 'react';
import { shallow } from 'enzyme';

import Header from './';

describe('Header', () => {
  const props = { apiKey: 'blah' };

  const element = shallow(<Header {...props} />);

  it('should render as expected', () => {
    expect(element).toMatchSnapshot();
  });
});
