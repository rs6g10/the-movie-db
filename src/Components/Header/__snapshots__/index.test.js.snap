// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render as expected 1`] = `
<div
  className="col-xs-12 header"
>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-sm-6 col-lg-3"
    >
      <Link
        replace={false}
        to="/"
      >
        <img
          alt="The Movie Database"
          className="logo"
          src="https://www.themoviedb.org/static_cache/v4/logos/stacked-green-cae7a95e2590dbdde28284ac26245cb2792788838f5c498b892e8d01c183e6f3.svg"
        />
      </Link>
    </div>
    <div
      className="col-xs-12 col-sm-6 col-lg-9"
    >
      <Searchbox
        apiKey="blah"
        placeHolder="Search"
        showResults={false}
      />
    </div>
  </div>
</div>
`;
